src/module.ts

```diff
@@ -37,24 +37,14 @@ export default defineNuxtModule<ModuleOptions>({
     }
   },
   defaults: {
-      publishableKey: process.env.STRIPE_PUBLISHABLE_KEY as string,
-      apiKey: process.env.STRIPE_API_KEY as string,
+      publishableKey: '' as string,
+      apiKey: '' as string,
       apiVersion: '2022-11-15'
   },
   setup (options, nuxt) {
     const { resolve } = createResolver(import.meta.url)
     const resolveRuntimeModule = (path: string) => resolveModule(path, { paths: resolve('./runtime') })
 
-    // Make sure the client key is set
-    if (!options.publishableKey) {
```

Comment:
Hum this would work for your use case for sure. But maybe would be better to have an option to enforce the non check if exist something like this. @fuentesloic  wdyt here this or we delete the check ?

Reply:
Some developers could decide to use only the client or the server side. Currently, our keys check is forcing devs to set both. 
So I think it's a good idea to remove this checker but I would replace it by limiting the usage of the stripe client or server service if not key defined. Then we should warn developers if they try to use the service with a missing key. wdyt ?

Reply:
> Currently, our keys check is forcing devs to set both.

Ah good point, this indeed is an important check. What do you think for the following:

- if both keys remain empty we could throw a non-blocking warning. (this would make build without keys possible)
- if only one key gets provided throw an blocking error. (like it is for the current state)
- if both keys are provided we are all good.

Do you (or stripe) have validation checks for the keys? We could include them in the process.

As soon as I'm able to I'll update it.

Reply:
I've been trying for a few hours but I'm not able to do so. But I've got to admit that this is my first time working on a nuxt module.

Reply:
Oh, it might have just been easier than I've though...

[useClientStripe.ts](https://github.com/Sandros94/nuxt-stripe/blob/612ad02c04452d3c1c48a5ffad5392288c431093/src/runtime/composables/useClientStripe.ts#L14-L19):
```ts
export default function useClientStripe() {
  const { public: { stripe: { publishableKey } } } = useRuntimeConfig()

  // Check status publishableKey
  if (!publishableKey) {
    throw new Error('Missing publishableKey option.')
  } else {
    console.info('Stripe publishableKey loaded successfully.');
  }

  return useNuxtApp().$stripe
}
```

[useServerStripe.ts](https://github.com/Sandros94/nuxt-stripe/blob/20bc622785ebb213992612a1e245a4e3bbd48cf1/src/runtime/server/services/useServerStripe.ts#L16-L21):
```ts
// Check status apiKey
  if (!apiKey) {
    throw new Error('Missing apiKey option.')
  } else {
    console.info('Stripe apiKey loaded successfully.');
  }
```

Reply:
Sounds like a good idea I was thinking the same @fuentesloic

Reply:
`throw new Error` might not be the best approach, while testing it prevents other tests to complete on build time for projects that would load the keys only at runtime.

This could probably due to #20

Reply:
Marking this as resolved, but need review below for [dc6b0f3](https://github.com/fuentesloic/nuxt-stripe/pull/19/commits/dc6b0f3039f045feefaf9041e677697aa1723100)